### DISCUSSION TITLE
Fine-tuning the dp model

### DISCUSSION THREAD

**Discussion by rajnichahal** [deepmodeling/deepmd-kit#2289] [General]
Hi,
I am trying to utilize the new finetuning feature as described here:
https://docs.deepmodeling.com/projects/deepmd/en/master/train/finetuning.html

However, 'finetuning' seems to be an unrecognized argument.

I am using the recent version of deepmd.

Thanks!

**Comment by njzjz** [comment]
Fine-tuning is available in the beta version [v2.2.0b0](https://github.com/deepmodeling/deepmd-kit/releases/tag/v2.2.0.b0).

**Reply by rajnichahal** [reply]
Oh ok. I will install that one. Thanks

**Reply by ZSHtju** [reply]
> Fine-tuning is available in the beta version [v2.2.0b0](https://github.com/deepmodeling/deepmd-kit/releases/tag/v2.2.0.b0).

I would like to ask that where I can get an offline package or conda install for v2.2.0b0? Currently, the only way is built from the source code.

**Comment by rajnichahal** [comment]
Hi @njzjz, I installed version 2.2.5 to try out the --finetune feature in deepmd. 
I noticed that in the deepmd documentation, the descriptor for pretrained model suitable for fine tuning is mentioned as 'se_atten'. But, I wanted to confirm if it is possible to finetune a pretrained model (descriptor type was se_e2_2) using additional data.

Please let me know. Thanks!

**Reply by njzjz** [reply]
Although the current code has checked the descriptor type, it is unclear if it is necessary (after #1592 is merged).

https://github.com/deepmodeling/deepmd-kit/blob/299184c71267074ebf00c019621c712ac2ed6c51/deepmd/utils/finetune.py#L43-L49

You can try to remove these lines and see if it works.

cc @iProzd @wanghan-iapcm

**Reply by rajnichahal** [reply]
Hi, I tested with DeePMD v2.2.5 and it gives me following error:
AssertionError: The finetune process only supports models pretrained with 'se_atten' or 'se_atten_v2' descriptor and 'ener' fitting_net!

Should I try with newest deepmd version, if it automatically allows finetuning on models pretrained with different descriptors?
Otherwise, I will try commenting these lines in v2.2.5 and see if it allows me to finetune.
Thanks!
